pkg/virtctl/vm/vm.go

```diff
@@ -528,6 +657,12 @@ func (o *Command) Run(args []string) error {
 		return addVolume(args[0], volumeName, namespace, virtClient, &dryRunOption)
 	case COMMAND_REMOVEVOLUME:
 		return removeVolume(args[0], volumeName, namespace, virtClient, &dryRunOption)
+	case COMMAND_EXPAND:
+		err = expandVirtualMachine(virtClient, namespace, args)
```

Comment:
I am not sure if its possible to do it like that, because `o` is defined like this
```golang
type Command struct {
	clientConfig clientcmd.ClientConfig
	command      string
}
```